keras/preprocessing/image.py

```diff
@@ -479,6 +479,16 @@ def __init__(self,
             raise ValueError('`zoom_range` should be a float or '
                              'a tuple or list of two floats. '
                              'Received arg: ', zoom_range)
+        if zca_whitening:
+            if not featurewise_center:
+                raise ValueError('This ImageDataGenerator specifies '
+                                 '`zca_whitening`, but does not '
+                                 'specify `featurewise_center`.')
+            if featurewise_std_normalization:
+                raise ValueError('This ImageDataGenerator specifies '
```

Comment:
Same, raise a warning saying that ZCA overrides `featurewise_std_normalization`, and set `featurewise_std_normalization` to False. This doesn't change the behavior as far as I can tell.